SpatialGDK/Source/SpatialGDK/Private/Utils/InterestFactory.cpp

```diff
@@ -382,4 +381,24 @@ QueryConstraint InterestFactory::CreateLevelConstraints() const
 	return LevelConstraint;
 }
 
+TArray<uint32> InterestFactory::CreateClientResultType() const
+{
+	TArray<uint32> ResultType;
+
+	// Add the required unreal components
+	ResultType.Add(SpatialConstants::UNREAL_METADATA_COMPONENT_ID);
```

Comment:
Hmm, don't know about listing components like this. I can see this easily being forgotten about when we add/remove components. Potentially a list of the GDK's WKC should exist somewhere in SpatialConstants? cc. @m-samiec @improbable-valentyn

Reply:
done